experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/agw-setup.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 
 - name: run magma deploy
-  command: ansible-playbook -e '@{{ usrHome }}/ansible-env.yaml' -e "magma_pkgrepo_dist=focal-{{ buildAgwPackage }}"  -i {{ usrHome }}/inventory {{ deployPath }}/magma_deploy.yml
+  command: ansible-playbook -e "magma_pkgrepo_dist=focal-{{ buildAgwPackage }}"  -i {{ usrHome }}/inventory {{ deployPath }}/magma_deploy.yml
```

Comment:
No need for the variables from ansible-env anymore? Are they being preset elsewhere?

Reply:
you deleted them in the previous PR, it's running fine without.

Reply:
I just checked, they were not even used in the role itself.

Reply:
I am looking at https://github.com/arunuke/magma/blob/f04e4ec03d794d756213b74134d2925d707524c5/experimental/cloudstrapper/playbooks/roles/magma-agw/tasks/agw-setup.yaml (which is the one where I added buildAgwPackage to agw-setup) and it still has the ansible-env.yaml file there. That said, if the build works fine even without that file, then that's great.